components/tools/src/main/java/edu/jhuapl/dorset/components/tools/CommandLineClient.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * 
+ */
+package edu.jhuapl.dorset.components.tools;
```

Comment:
OK - I updated my default Eclipse code templates to include this copyright for newly created classes, as well as removed the author field by default. Should this modified code template be provided as part of the project in addition to the style guide?